src/components/Footer/Footer.jsx

```diff
@@ -5,9 +5,9 @@ import logo from "../../assets/logo.svg";
 
 import { Link } from "react-router-dom";
 
-import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
-import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
-import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";
+import FacebookIcon from "../../assets/icon-facebook.svg";
+import PinterestIcon from "../../assets/icon-pinterest.svg";
+import TwitterIcon from "../../assets/icon-twitter.svg";
```

Comment:
Please this needs to remain the same 
```suggestion
import { ReactComponent as FacebookIcon } from "../../assets/icon-facebook.svg";
import { ReactComponent as PinterestIcon } from "../../assets/icon-pinterest.svg";
import { ReactComponent as TwitterIcon } from "../../assets/icon-twitter.svg";